Dockerfile

```diff
@@ -1,5 +1,7 @@
-# Use the official Node.js 18 image as the base image for building the plugins
-FROM node:18@sha256:d0bbfdbad0bff8253e6159dcbee42141db4fc309365d5b8bcfce46ed71569078 AS builder
+ARG BASE_IMAGE_VERSION=18
+ARG FINAL_IMAGE_VERSION=3.20.3
```

Comment:
For security reasons we need the full image version with the hash.

Reply:
> For security reasons we need the full image version with the hash.

Would you like to get the [dependabot](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference?learn=dependency_version_updates&learnProduct=code-security) , which could watch updates and make a PR for the updated docker image versions?

Reply:
I thought we had it. I will check it out. But in the current code, we need to keep the hash.